WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsSelectPlugin.swift

```diff
@@ -65,7 +67,9 @@ struct InPersonPaymentsSelectPlugin: View {
             Spacer()
 
             Button(Localization.confirm) {
```

Comment:
I'm still finding my way with SwiftUI so I had a bit of a play here.

What do you think of this tweak?

`Button(Localization.confirm, action: confirmPluginSelection)`

```
private extension InPersonPaymentsSelectPluginView {
    func confirmPluginSelection() {
        if let selectedPlugin = selectedPlugin {
            onPluginSelected(selectedPlugin)
        }
    }
}
```

Reply:
I think both options are fine. Generally, I lean towards inlining things in SwiftUI unless the view is getting long and complex, but I think this one is still small enough. I might switch it just because the optional unwrapping feels a bit out of place there